### ISSUE TITLE
SA1101 should not warn here

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/StyleCopAnalyzers#2799]
```cs
class C
{
    public C()
        : this(nameof(↓P))
    {
    }

    public C(string p)
    {
        this.P = p;
    }

    public string P { get; }
}
```

Code fix generates broken code.
